### DISCUSSION TITLE
Network separation Interface Web/Application

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#2774] [General] [answered]
Hello, i try to find something but didn't find my need on the documentation.

I would like to separate the administration/application
like : 
- eth0 for the web interface for all product
- eth1 to received the logs, wazuh agent communication, etc.. did the product get this configuration in a menu or i have to modify the system myself?

Thank you :)

**Comment by weslambert** [comment, marked as answer]
We don't currently support separating the management interface(s) as you've described.

**Reply by b4b857f6ee** [reply]
ok thank you.
